### ISSUE TITLE
Standardize Buffer size when doing any sort of IPC

### ISSUE THREAD

**Issue by hydralauncher** [hydralauncher/hydra#111]

**Comment by JoaoVictor6** [comment]
I can solve this issue!

Can you give me more details?

**Comment by hydralauncher** [comment]
@JoaoVictor6 of course. Basically, the Python process and the Node.js process have no standard amount of bytes when communicating in between each other. To solve this, instead of attaching Node.js to listen to the event, we can create a while loop that ticks a .read() on the socket always reading a fixed amount of bytes (1024 * 2 for instance).

The I/O pipe operations for the Python process occurs here:
https://github.com/hydralauncher/hydra/blob/main/torrent-client/fifo.py

The I/O pipe operations for the Node.js process occurs here:
https://github.com/hydralauncher/hydra/blob/main/src/main/services/fifo.ts
https://github.com/hydralauncher/hydra/blob/main/src/main/main.ts#L51
